esmvaltool/cmorizers/data/downloaders/datasets/esacci_fire.py

```diff
@@ -0,0 +1,35 @@
+"""Script to download ESACCI-FIRE."""
+
+from dateutil import relativedelta
+
+from esmvaltool.cmorizers.data.downloaders.ftp import CCIDownloader
+
+
+def download_dataset(config, dataset, start_date, end_date, overwrite):
+    """Download dataset.
+
+    Parameters
+    ----------
+    config : dict
+        ESMValTool's user configuration
+    dataset : str
+        Name of the dataset
+    start_date : datetime
+        Start of the interval to download
+    end_date : datetime
+        End of the interval to download
+    overwrite : bool
+        Overwrite already downloaded files
+    """
+    loop_date = start_date
+
+    downloader = CCIDownloader(config=config,
+                               dataset=dataset,
+                               overwrite=overwrite)
+    downloader.connect()
+
+    downloader.set_cwd('burned_area/MERIS/grid/v4.1/')
+    while loop_date <= end_date:
+        year = loop_date.year
+        downloader.download_year(f'{year}')
```

Comment:
This downloader is not working for me. Looking at the ftp structure of the source, I think the year is missing in the directory path to the data, for example: `ftp://anon-ftp.ceda.ac.uk/neodc/esacci/fire/data/burned_area/MERIS/grid/v4.1/2006/20060207-ESACCI-L4_FIRE-BA-MERIS-fv4.1.nc`

Reply:
It is working fine for me. Can you please test again?